site/docs/pages/build-your-first-onchain-app.mdx

```diff
@@ -0,0 +1,72 @@
+
+# Build your first Onchain app
+
+Build your first Onchain app with OnchainKit's App Template. 
+
+Play with it live [here](https://onchain-app-template.vercel.app).
+
+<img
+  alt="Onchain app template"
+  src="https://onchainkit.xyz/assets/Onchain-app-template-1.png"
+  width="auto"
+/>
+
+::::steps
+
+### Install
```

Comment:
I don't think we need this step, as the developer will:
1. click `use this template` or `fork`
2. run `bun i`, which will take care of installing OnchainKit

Reply:
Cool!